### DISCUSSION TITLE
NeqSim documentation

### DISCUSSION THREAD

**Discussion by EvenSol** [equinor/neqsim#194] [General]
You find documentation on the [NeqSim homepage](https://equinor.github.io/neqsimhome/), the [Java Wiki](https://github.com/equinor/neqsim/wiki), the [Python Wiki](https://github.com/equinor/neqsimpython/wiki) and the [Matlab Wiki](https://github.com/equinor/neqsimmatlab/wiki).

**Comment by kjensemann** [comment]
Nå er jeg i gang. Har fått tilgang til neqsim-biblioteket i Android nå, så skal prøve å lage en liten UMR flashe-app her, for å bli litt kjent med objektmodellen. 

Kule greier så langt! 

![image](https://github.com/equinor/neqsim/assets/38838655/f28b8a67-8708-491d-9904-ebb5fd8ef6ae)
